src/derived.rs

```diff
@@ -150,7 +161,10 @@ where
     }
 
     fn durability(&self, db: &DB, key: &Q::Key) -> Durability {
-        self.slot(key).durability(db)
+        match self.get_slot(key) {
+            Some(slot) => slot.durability(db),
+            None => Durability::LOW,
```

Comment:
It's non-obvious why this `::LOW` is correct. I think the reason why it is correct is that we actually don't care, as it is only used for debug. Perhaps we should rename this method to `debug_durability`?